### PR TITLE
Add .up_to_infinity()

### PR DESCRIPTION
**Pull request by ThinkChaos**
Counts up to a given type's maximum value.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -67,6 +67,20 @@ impl<Idx> From<Idx> {
     }
 }
 
+impl<Idx> From<Idx>
+where
+    Idx: Bounded,
+{
+    /// Construct a [Range] that counts up to `Idx`'s maximum value.
```
Comment (killercup) [inline src/lib.rs:74]: Can you put the test from below here as doc test?

**Review by killercup — COMMENTED** [review]
Thanks, this looks great! Just one small note and then it's good to merge :)

~~BTW, feel free to add "Fixes #2" to the commit message or description!~~ ~~Ah it just didn't show up on mobile, sorry!~~ You wrote "Solves #2" which will not actually close the issue. Was this on purpose? You are welcome to change it to fixes, as this PR does indeed solve the issue :)

**Comment by ThinkChaos** [comment]
Also changed "solves" to "fixes"

**Review by killercup — APPROVED** [review]
Perfect, thanks!

**Pull request merged**